src/popover.ts

```diff
@@ -20,6 +20,16 @@ export function isSupported() {
   );
 }
 
+export function isPolyfilled() {
+  // if the `showPopover` method is defined but is not "native code"
+  // then we can infer it's been polyfilled
+  return (
+    document.body &&
+    document.body.showPopover &&
+    !document.body.showPopover.toString().match(/native code/i)
```

Comment:
I am not particularly attached to this implementation, so if you have suggestions or think this code could/should be written in a different way, just let me know.